### PR TITLE
made log in and sign in page similar to homepage ==

### PR DESCRIPTION
**Pull request by tieuu**
changed some flash and alert
log in and sign in now look like they're in the same website with home page ;_;
redirect to equipment index when user is signed in instead of the home page (which leads to log in and sign up lol)

### CODE REVIEW

File: app/assets/stylesheets/pages/_index.scss [hunk 1]
```diff
@@ -1,2 +1,2 @@
 // Import page-specific CSS files here.
-@import "home";
+
```
Comment (JulianS61) [inline app/assets/stylesheets/pages/_index.scss]: why did you delete this line ?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -10,5 +10,4 @@
 
 // Your CSS partials
 @import "components/index";
-@import "pages/index";
 @import "pages/home";
```
Comment (JulianS61) [inline app/assets/stylesheets/application.scss]: keep index, delete home ?

Comment (tieuu) [reply app/assets/stylesheets/application.scss]: because in home there's only one line "@import index"..., if we need to do sth with that later i will add :|
